pkg/reconciler/inmemorychannel/dispatcher/controller.go

```diff
@@ -56,8 +61,32 @@ const (
 	writeTimeout  = 15 * time.Minute
 	port          = 8080
 	finalizerName = "imc-dispatcher"
+
+	readinessProbeOK       = http.StatusNoContent
```

Comment:
`OK` for me implies `200 OK`, while `No Content` translates to 204 :shrug:

Reply:
Isn't 204 like 200 but without body?
Would you suggest to return 200 with a body like OK?

Reply:
`readinessProbeNoConent` ?

The name `...OK` is imo wrong… since we do not do „OK“. We do „NoContent“

Reply:
Replaced `...OK` with `...Ready` which may refer to any of 20x response codes. I hope it suits everyone :slightly_smiling_face: